web-client/src/presenter/actions/StatusReportOrderResponse/prepareStatusReportOrderResponseAction.ts

```diff
@@ -0,0 +1,84 @@
+import { FORMATS } from '@shared/business/utilities/DateHandler';
+import { state } from '@web-client/presenter/app.cerebral';
+
+export const prepareStatusReportOrderResponseAction = ({
+  applicationContext,
+  get,
+  store,
+}: ActionProps) => {
+  const {
+    additionalOrderText,
+    dueDate,
+    issueOrder,
+    jurisdiction,
+    orderType,
```

Comment:
Any chance we can use the ORDER_TYPES constant, or a subset of the fields in those objects?

Reply:
I'm not sure. For now, I have at least added a constant.